### PR TITLE
Make trap stones visuallity less obvious using colorize

### LINKED ISSUE #984: Use texture mods for ctf_trap textures
All of the unwalkable nodes can use something like [colorize
There were also requests to make the overlays a little less obvious, that could be done too

### PR DESCRIPTION
**Pull request by farooqkz**
Screenshots attached
screenshot_20220724_160155
screenshot_20220724_160255

Fixes #984

### CODE REVIEW

**Comment by LoneWolfHT** [comment]
Please disable texturepacks when showing texture changes
image

File: mods/ctf/ctf_map/ctf_traps.lua [hunk 1]
```diff
@@ -36,6 +36,14 @@ minetest.register_node("ctf_map:spike", {
...
 		fixed = {-0.5, -0.5, -0.5, 0.5, 0, 0.5},
 	},
+	after_place_node = function(pos, placer, itemstack, pointed_thing)
+		local meta = minetest.get_meta(pos)
+		local name = placer:get_player_name()
+
+		meta:set_string("placer", minetest.serialize({
+			name = name
+		}))
+	end
```
Comment (LoneWolfHT) [inline mods/ctf/ctf_map/ctf_traps.lua:46]: What is this bit for?

Comment (farooqkz) [reply mods/ctf/ctf_map/ctf_traps.lua:46]: oh sorry it shouldn't be there

**Comment by farooqkz** [comment]
> Please disable texturepacks when showing texture changes image

why? isn't the point of `[colorize` is making it less obvious for users of texture packs?

**Comment by LoneWolfHT** [comment]
> > Please disable texturepacks when showing texture changes image
> 
> why? isn't the point of `[colorize` is making it less obvious for users of texture packs?

You can do both if you want, but I have no idea what I'm looking at with non-default textures

**Review by LoneWolfHT — CHANGES_REQUESTED** [review]
Please change the unwalkable node [colorize overlays to yellow (to match color of the current textures) and revert the changes to the damage cobble, we'll leave that obvious for now

**Comment by farooqkz** [comment]
> Please change the unwalkable node [colorize overlays to yellow (to match color of the current textures) and revert the changes to the damage cobble, we'll leave that obvious for now

Done. Screenshot attached.

**Review by LoneWolfHT — APPROVED** [review]

**Pull request merged**